earth_enterprise/src/common/khCppStd.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Google Inc.
```

Comment:
This should say `Copyright 2018 the OpenGEE Contributors`.  I don't think Google is contributing to this project any longer.

Reply:
Good catch!  And thanks for all the feedback!  I'm going fix these things up and do another push.